src/components/MenuItem.js

```diff
@@ -75,7 +75,7 @@ const MenuItem = (props) => {
             }}
             style={({hovered, pressed}) => ([
                 styles.popoverMenuItem,
-                StyleUtils.getButtonBackgroundColorStyle(getButtonState(props.focused || hovered, pressed, props.success, props.disabled, props.interactive)),
+                StyleUtils.getButtonBackgroundColorStyle(getButtonState(props.focused || hovered, pressed, props.success, props.disabled, props.interactive), true),
```

Comment:
Hover background colour for menu item turned out to be the same as the one for block quotes thereby causing [this issue](https://github.com/Expensify/App/issues/41901) where the quote line became invisible.